inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -87,6 +99,12 @@ struct jit_uni_roi_pooling_kernel_f32 : public jit_uni_roi_pooling_kernel, publi
     Xmm xmm_xf = Xmm(1);
     Vmm vmm_xf = Vmm(1);
 
+    std::unique_ptr<jit_load_emitter> load_emitter = nullptr;
+    std::unique_ptr<jit_store_emitter> store_emitter = nullptr;
+
+    std::vector<size_t> store_pool_gpr_idxs;
```

Comment:
These pools are supposed to be passed to load_emmiters, but at the moment they are not used and ignored in current emitters implementation.
Will be discussed.